src/components/siteContent/SiteContent.js

```diff
@@ -58,8 +47,13 @@ const SiteContent = ({ group1, group2, group3 }) => {
                 onMouseOver={onPhotoHover}
                 onMouseLeave={onMouseLeave}
               />
-              {isDescriptionOpened && index === Number(isPhotoHovered) && (
-                <div className={styles.description}>{isPhotoHovered}</div>
+              {isDescriptionOpened && index === Number(hoveredPhoto) && (
+                <div className={styles.description}>
+                  <p style={{ fontWeight: "bold" }}>
```

Comment:
raz robisz inline, raz w module, od czego zależy kiedy tu a kiedy tu?

Reply:
generalnie w module, a jak chce nadpisać w jakimś jednym miejscu styl no to wale inline

Reply:
wrzucone do module :)